pkg/runtime/k0s/utils.go

```diff
@@ -74,3 +74,14 @@ func FetchKubeconfigAndGetKubectl(ssh ssh.Interface, host net.IP, rootfs string)
 	}
 	return nil
 }
+
+func (k *Runtime) WaitK0sReady(ssh ssh.Interface, host net.IP) error {
+	bytes, err := ssh.Cmd(host, "k0s status")
```

Comment:
What will happen if the remote command `k0s status` hangs? Shall we take this into consideration?
Add a timeout policy? @starComingup

Reply:
After I learned k0s source code, `k0s status` will return a process ID and throw Error, It will not hang on.